librocksdb_sys/src/lib.rs

```diff
@@ -20,7 +20,7 @@ extern crate tempdir;
 use libc::{c_char, c_uchar, c_int, c_void, size_t, uint8_t, uint32_t, uint64_t, c_double};
 use std::ffi::CStr;
 
-pub enum DBOptions {}
+pub enum DBOption {}
```

Comment:
cause I divided the Options into DBOptions and ColumnFamilyOptions. I think it will be a little bit confused with this "DBOptions"